src/chart/rendering/items/drawitem.cpp

```diff
@@ -101,10 +136,50 @@ Geom::Line DrawItem::getLabelPos(
 	return Geom::Line(res);
 }
 
+bool DrawItem::bounds(const Geom::Point &point)
+{
+	if ((double)enabled == 0) return false;
```

Comment:
Static cast

Reply:
Maybe we should eliminate all occurances then switch on -Wold-style-cast

Reply:
replaced